sdks/java/core/src/main/java/org/apache/beam/sdk/testing/StaticWindows.java

```diff
@@ -100,9 +102,22 @@ public boolean isCompatible(WindowFn<?, ?> other) {
   }
 
   @Override
-  public BoundedWindow getSideInputWindow(BoundedWindow window) {
-    checkArgument(windows.get().contains(window),
-        "StaticWindows only supports side input windows for main input windows that it contains");
-    return window;
+  public WindowMappingFn<BoundedWindow> getDefaultWindowMappingFn() {
```

Comment:
Surprised StaticWindows isn't parameterized... Should it be (before making the change is backwards incompatible)?